### ISSUE TITLE
choose best vm size

### ISSUE THREAD

**Issue by bogsnork** [bogsnork/nectarvision#2]
trialling multiple object classification.  Tried on N6 vm but interrupted after 30 mins.  Now on N24 vm.  Run time seems similar.  Here's a message from the console which may help interpret: 
code:
`#define model ----
feature_extractor <-
  application_xception(
    include_top = FALSE,
    input_shape = c(224, 224, 3),
    pooling = "avg"
  )`
console:
`2019-11-29 12:53:43.323773: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2019-11-29 12:53:43.714133: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 0 with properties: 
name: Tesla K80 major: 3 minor: 7 memoryClockRate(GHz): 0.8235
pciBusID: 49a0:00:00.0
totalMemory: 11.17GiB freeMemory: 11.10GiB
2019-11-29 12:53:43.840109: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 1 with properties: 
name: Tesla K80 major: 3 minor: 7 memoryClockRate(GHz): 0.8235
pciBusID: 65dd:00:00.0
totalMemory: 11.17GiB freeMemory: 11.10GiB
2019-11-29 12:53:43.967950: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 2 with properties: 
name: Tesla K80 major: 3 minor: 7 memoryClockRate(GHz): 0.8235
pciBusID: 82ff:00:00.0
totalMemory: 11.17GiB freeMemory: 11.10GiB
2019-11-29 12:53:44.098963: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1432] Found device 3 with properties: 
name: Tesla K80 major: 3 minor: 7 memoryClockRate(GHz): 0.8235
pciBusID: a4d9:00:00.0
totalMemory: 11.17GiB freeMemory: 11.10GiB
2019-11-29 12:53:44.099340: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1511] Adding visible gpu devices: 0, 1, 2, 3
2019-11-29 12:53:56.532364: I tensorflow/core/common_runtime/gpu/gpu_device.cc:982] Device interconnect StreamExecutor with strength 1 edge matrix:
2019-11-29 12:53:56.532436: I tensorflow/core/common_runtime/gpu/gpu_device.cc:988]      0 1 2 3 
2019-11-29 12:53:56.532449: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 0:   N N N N 
2019-11-29 12:53:56.532456: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 1:   N N N N 
2019-11-29 12:53:56.532463: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 2:   N N N N 
2019-11-29 12:53:56.532470: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1001] 3:   N N N N 
2019-11-29 12:53:56.533276: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10759 MB memory) -> physical GPU (device: 0, name: Tesla K80, pci bus id: 49a0:00:00.0, compute capability: 3.7)
2019-11-29 12:53:56.553275: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:1 with 10759 MB memory) -> physical GPU (device: 1, name: Tesla K80, pci bus id: 65dd:00:00.0, compute capability: 3.7)
2019-11-29 12:53:56.553616: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:2 with 10759 MB memory) -> physical GPU (device: 2, name: Tesla K80, pci bus id: 82ff:00:00.0, compute capability: 3.7)
2019-11-29 12:53:56.554337: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1115] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:3 with 10759 MB memory) -> physical GPU (device: 3, name: Tesla K80, pci bus id: a4d9:00:00.0, compute capability: 3.7)`

**Comment by bogsnork** [comment]
well, that was quite fast on NC24 vm: 

Epoch 1/20
164/164 [==============================] - 620s 4s/step - loss: 0.6729 - acc: 0.6714 - val_loss: 0.2383 - val_acc: 0.9452
Epoch 2/20
164/164 [==============================] - 533s 3s/step - loss: 0.2444 - acc: 0.9205 - val_loss: 3.5044 - val_acc: 0.3548
Epoch 3/20
164/164 [==============================] - 525s 3s/step - loss: 0.1769 - acc: 0.9445 - val_loss: 0.2475 - val_acc: 0.9433
